packages/core-data/src/hooks/use-entity-records.ts

```diff
@@ -104,3 +89,16 @@ export default function __experimentalUseEntityRecords< RecordType >(
 		...rest,
 	};
 }
+
+export function __experimentalUseEntityRecords(
```

Comment:
Is the full signature necessary to make TS happy? Otherwise, it would be perfectly fine to use `...params` in both places.

Reply:
That's what I tried, but TS wants to see the same number of arguments in both functions and using `...params` makes it 4 arguments vs a variable number of arguments.